fabtests/Makefile.am

```diff
@@ -140,6 +140,7 @@ nobase_dist_config_DATA = \
 	pytest/default/test_ubertest.py \
 	pytest/default/test_unexpected_msg.py \
 	pytest/efa/conftest.py \
+	pytest/efa/efa_common.py \
```

Comment:
The file itself is missing.

Reply:
Thank you! Forgot to add it.

Fixed in new revision.